commit f9c3e57ac25511db78f3d51a38f6a715be220479
Author: Adhemerval Zanella <adhemerval.zanella@linaro.org>
Date:   Tue May 31 12:22:13 2022 -0300

    posix: Use 64 bit stat for fpathconf (_PC_ASYNC_IO) (BZ# 29208)
    
    This is a missing spot initially from 52a5fe70a2c77935.
    
    Checked on i686-linux-gnu.
    
    (cherry picked from commit 6e7137f28c9d743d66b5a1cb8fa0d1717b96f853)

diff --git a/sysdeps/posix/fpathconf.c b/sysdeps/posix/fpathconf.c
index ec0e780466756e00..e673f2016136679e 100644
--- a/sysdeps/posix/fpathconf.c
+++ b/sysdeps/posix/fpathconf.c
@@ -131,9 +131,9 @@ __fpathconf (int fd, int name)
 #ifdef	_POSIX_ASYNC_IO
       {
 	/* AIO is only allowed on regular files and block devices.  */
-	struct stat64 st;
+	struct __stat64_t64 st;
 
-	if (__fstat64 (fd, &st) < 0
+	if (__fstat64_time64 (fd, &st) < 0
 	    || (! S_ISREG (st.st_mode) && ! S_ISBLK (st.st_mode)))
 	  return -1;
 	else
